src/feature_scaling.jl

```diff
@@ -87,6 +87,10 @@ function center!(D::AbstractDataFrame, colnames::AbstractVector{Symbol})
     for colname in colnames
         if eltype(D[colname]) <: Real
             μ = mean(D[colname])
+            if isna(μ)
+                warn("Column $colname contains NA values, skipping rescaling of this column!")
```

Comment:
Just a little nitpick. maybe we should use some quotes `"Column \"$colname\" contains ..."`